### ISSUE TITLE
Create a separate new page to see the available contents. 

### ISSUE THREAD

**Issue by mohtasheem135** [GDG-On-Campus-ICTU/MaterialHub-Hactoberfest#15]
I can create a separate page for the available contest to view the data properly. 

Please assign this task to me and the label of hactoberfest and gaoc-extd

**Comment by Perry-Bright** [comment]
I think it's best to use split screen since the aim is to create a one-page webapp

**Comment by mohtasheem135** [comment]
Okay I understand, but what if there is more data the users have to keep on scrolling we must think of something to sort and filter.

**Comment by Perry-Bright** [comment]
Exactly, filtering by tags will be very great along with some advanced search function accounts for partial matches or typos.
